app/src/main/java/eu/kanade/tachiyomi/data/backup/models/BackupChapter.kt

```diff
@@ -21,6 +21,8 @@ data class BackupChapter(
     @ProtoNumber(9) var chapterNumber: Float = 0F,
     @ProtoNumber(10) var sourceOrder: Long = 0,
     @ProtoNumber(11) var lastModifiedAt: Long = 0,
+    @ProtoNumber(12) var version: Long = 0,
+    @ProtoNumber(13) var isSyncing: Long = 0,
```

Comment:
Can these backup numbers be PRd to Mihon? I don't want to create accidental conflicts

Reply:
Done opened PR:   https://github.com/mihonapp/mihon/pull/450